examples/rds_privatelink_setup/README.md

```diff
@@ -50,15 +50,18 @@ This example demonstrates how to create a new Amazon RDS Postgres instance and c
     module "materialize_privatelink_rds" {
         source = "../.."
 
-        mz_rds_instance_name = var.mz_rds_instance_name
-        mz_rds_vpc_id        = module.rds_postgres.vpc.vpc_id
-        aws_region           = var.aws_region
+        mz_rds_instance_details = [
+            { name = "instance1", listener_port = 5001 },
+            { name = "instance2", listener_port = 5002 }
```

Comment:
a comment about the uniqueness requirement on listener ports should go somewhere. I think here?